test/client.test.js

```diff
@@ -103,21 +103,6 @@ test('should use custom strategy', t => {
     t.true(result);
 });
 
-test('should use a set of custom strategies', t => {
```

Comment:
Why is this test not needed?

Reply:
Duplicate. Ava throws on tests with same title (why I changed the title of some others)